src/test/java/com/mabl/integration/jenkins/MablStepDeploymentRunnerTest.java

```diff
@@ -79,7 +79,7 @@ public void setup() {
         );
     }
 
-    @Test
+    @Test(timeout = 6000000)
```

Comment:
Should we really give this test 1 2/3 hours to run?

Reply:
Ooops! That's a mistake.